frontend/src/localization/en.json

```diff
@@ -90,7 +90,7 @@
     "resetSettings": "Reset all settings",
     "excludeTimestamps": "Exclude from time calculation",
     "excludingTimestamps": "Excluded from time calculation",
-    "excludeFromSecondaryIdQuery": "Exclude from analysis layer",
+    "excludeFromSecondaryId": "Exclude from analysis layer",
     "excludingFromSecondaryIdQuery": "Excluded from analysis layer",
```

Comment:
Fine with me, but could you also rename all `excludingFromSecondaryIdQuery` to `excludingFromSecondaryId` then?